### PR TITLE
feat: info-log webhooks domain part of url

### PR DESCRIPTION
**Pull request by daveleek**
Logging the domain part of the webhook url.
An effort to gain more knowledge around what type of integrations we would like to focus on

### CODE REVIEW

File: src/lib/services/addon-service.ts [hunk 1]
```diff
@@ -259,6 +275,23 @@ export default class AddonService {
...
+            addonConfig.provider === 'webhook' &&
+            addonConfig.parameters.url &&
+            addonConfig.parameters.url.indexOf('://') > 0
+        ) {
+            const start = addonConfig.parameters.url.indexOf('://') + 3;
+            let end = addonConfig.parameters.url.indexOf('/', start);
+            if (end === -1) {
+                end = addonConfig.parameters.url.length;
+            }
+            const domain = addonConfig.parameters.url.substring(start, end);
```
Comment (gastonfournier) [inline src/lib/services/addon-service.ts:289]: Duplicated code, can't this be extracted to a function?

**Review by gastonfournier — COMMENTED** [review]
I think it'd be ideal to log the domain every time the webhook is called, not when created or modified.

File: src/lib/services/addon-service.ts (hunk 1 above)
Comment (sighphyre) [reply src/lib/services/addon-service.ts:289]: But also... why not just?
```
new URL(addonConfig.parameters.url).hostname
```

Comment (daveleek) [reply src/lib/services/addon-service.ts:289]: Oh nice, did not even think about that. But obviously this exists 🙄

**Comment by daveleek** [comment]
Closing this and redoing

**Pull request closed without merging**